app/controllers/admin/genres_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Admin::GenresController < Admin::BaseController
+  def create
+    genre = Genre.new(genre_params)
+    flash[:notice] = 'Oops, you didn\'t include the genre name' unless genre.save
```

Comment:
@AliSchlereth, I think what you meant by improving the UI if the genre didn't save was make it so there's more than JUST the flash message if creation fails... Either way, I wanted the user to be back on the genres index page (I thought), so I just made it so everybody redirects to the index, but the flash appears for failure to create. Please let me know if I interpreted that incorrectly.

Reply:
Yep, this works!